src/app/store/actionCreators/gameActions.ts

```diff
@@ -1,6 +1,11 @@
-import { GameActionTypes, TogglePauseAction } from '../types';
+import { GameActionTypes, RestartAction, TogglePauseAction } from '../types';
 
 export const togglePause = (isPause: boolean): TogglePauseAction => ({
   type: GameActionTypes.TOGGLE_PAUSE,
   payload: !isPause,
 });
+
+export const restart = (isRestart: boolean): RestartAction => ({
+  type: GameActionTypes.RESTART,
+  payload: isRestart,
```

Comment:
Could you explain in short words why do we need boolean payload?

Reply:
For handle when user pressed the restart button